### ISSUE TITLE
[REST] Ważność zdjęć

### ISSUE THREAD

**Issue by SlawomirMizerski** [allegro/allegro-api#526]
Jeśli ważność zdjęć minie a zdjęcia są wykorzystywane w aukcji to nie są usuwane ale czy kiedy aukcja zostanie zakończona to czy zdjęcia są od razu usuwane i podczas wznawiania aukcji potrzebne będzie wgranie ponownie zdjęć?

**Comment by KrzysztofLukanowski** [comment]
Nie, nie usuwamy zdjęć od razu po zakończeniu oferty. Zdjęcia znikną wraz z przeniesieniem oferty do archiwum, czyli po upływie 60 dni od zakończenia.

**Comment by matee77** [comment]
Ile czasu jest ważne zdjęcie wykorzystane w drafcie nieaktywowanej oferty? Czy jest określony czas, po którym nieaktywowany draft i linki do wrzuconych zdjęć tracą ważność?

**Comment by KrzysztofLukanowski** [comment]
Termin ważności to 60 dni od momentu ostatniego, w którym oferta otrzymała status inactive/ended.
